mockserver/mockserver_test.go

```diff
@@ -4,17 +4,20 @@ import (
 	"context"
 	"testing"
 
+	"github.com/btcsuite/btcd/btcjson"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/filecoin-project/go-jsonrpc"
 	"github.com/stretchr/testify/assert"
 )
 
 type Client struct {
```

Comment:
Let's define the client outside of tests - maybe the `client` directory.

Reply:
@robert-zaremba moved client to it's own directory, please check.